boards/arm/rcar_h3ulcb/doc/rcar_h3ulcb.rst

```diff
@@ -122,60 +130,111 @@ Here is CN04 UART interface pinout (depending on your Kingfisher board version)
 | GND    | 9        | 6        |
 +--------+----------+----------+
 
+CAN :
+-----
+
+H3ULCB board is providing two CAN interfaces. Both interfaces are available on the Kingfisher daughter board.
+
++--------------------+--------------------+--------------+
+| Physical Interface | Software Interface | Transceiver  |
++====================+====================+==============+
+| CN17               | CAN0               | TCAN332GDCNT |
++--------------------+--------------------+--------------+
+| CN18               | CAN1               | TCAN332GDCNT |
++--------------------+--------------------+--------------+
+
+.. Note:: Interfaces are set to 125 kbit/s by default.
+
+Here is CAN physical interfaces pinout :
+
++-----+--------+
+| Pin | Signal |
++=====+========+
+| 1   | CANH   |
++-----+--------+
+| 2   | CANL   |
++-----+--------+
+| 3   | GND    |
++-----+--------+
+
+I2C :
+-----
+
+H3ULCB board is providing two I2C buses. Unfortunately current support doesn't provide direct access to these buses through connectors.
+
+I2C is mainly used to manage and power-on a lot of onboard chips on the H3ULCB and Kingfisher daughter board.
+
+All these devices are connected to I2C buses through I2C mux and I/O expanders that are not yet supported with this board.
```

Comment:
What "all these devices" mean?

Reply:
I meant "chips" from above sentence

Reply:
Can sentence be rephrased to be more clear?

Reply:
Yes, it is better like that ?
"Embedded I2C devices and I/O expanders are not yet supported. The current I2C support therefore does not make any devices available to the user at this time."

Reply:
Yes, sounds better, thanks!